### PR TITLE
Correctly import NullCache class

### PR DESCRIPTION
**Pull request by nickvergessen**
Caused by #16869

@LukasReschke @MorrisJobke @DeepDiver1975

### CODE REVIEW

File: tests/lib/server.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+	/**
+	 * @dataProvider dataTestQuery
+	 *
+	 * @param string $serviceName
+	 * @param string $instanceOf
+	 */
+	public function testQuery($serviceName, $instanceOf) {
+		$this->assertInstanceOf($instanceOf, $this->server->query($serviceName), 'Service ' . $serviceName . ' did not return the right class');
+	}
```
Comment (LukasReschke) [inline tests/lib/server.php]: Cool idea :rocket:

**Comment by nickvergessen** [comment]
Will add tests for the other services as well.

File: tests/lib/server.php (hunk 1 above)
Comment (MorrisJobke) [inline tests/lib/server.php]: You rock :rocket:

**Comment by DeepDiver1975** [comment]
THX a lot!

**Comment by nickvergessen** [comment]
Pushed, ready to review

**Comment by nickvergessen** [comment]
```
There was 1 error:
12:13:58 
12:13:58 1) Test\Server::testGetAppFolder
12:13:58 Exception: The requested uri() cannot be processed by the script '/usr/local/bin/phpunit')
12:13:58 
12:13:58 /var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/lib/private/appframework/http/request.php:559
12:13:58 /var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/lib/private/appframework/http/request.php:585
12:13:58 /var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/lib/private/app.php:694
12:13:58 /var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/lib/private/server.php:559
12:13:58 /var/jenkins/workspace/pull-request-analyser-ng-simple/label/SLAVE/tests/lib/server.php:154
```

Can someone reproduce the problem locally?

**Comment by jancborchardt** [comment]
Master works again for me, as I ran into the error before.

**Comment by MorrisJobke** [comment]
:+1:

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**